ocs_ci/framework/conf/fusion_version/fusion-2.10.yaml

```diff
@@ -0,0 +1,11 @@
+DEPLOYMENT:
+  fusion_deployment: True
+  fdf_deployment: True
+  channel: v2.0
```

Comment:
@vavuthu , can we have some prefix to the ```channel``` ? because in all the config files we have prefixes like ```ocp_channel```, ```acm_hub_channel``` etc.

Reply:
make sense. took it from base fusion yaml which comes with channel. will make chnages in follow-up PR ( need to check in functionality where we are using channel )